### PR TITLE
Client mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/client/particle/Particle.mapping [hunk 1]
```diff
@@ -7,9 +7,9 @@ CLASS dik net/minecraft/client/particle/Particle
 	FIELD b prevPosX D
 	FIELD c prevPosY D
 	FIELD d prevPosZ D
```
Comment (liach) [inline mappings/net/minecraft/client/particle/Particle.mapping:9]: `prevX`, etc.

File: mappings/net/minecraft/client/search/IdentifierSearchableContainer.mapping [hunk 2]
```diff
@@ -5,4 +5,7 @@ CLASS dwr net/minecraft/client/search/IdentifierSearchableContainer
 	FIELD c identifierFinder Ljava/util/function/Function;
 	FIELD d entries Ljava/util/List;
 	FIELD e entryIds Lit/unimi/dsi/fastutil/objects/Object2IntMap;
+	METHOD a compare (Ljava/lang/Object;Ljava/lang/Object;)I
+		ARG 1 object1
+		ARG 2 object2
```
Comment (liach) [inline mappings/net/minecraft/client/search/IdentifierSearchableContainer.mapping:10]: Just curious, how does this actually compare objects?

Comment (Runemoro) [reply mappings/net/minecraft/client/search/IdentifierSearchableContainer.mapping:10]:
```java
    protected int compare(T object1, T object2) {
        return Integer.compare(this.entryIds.getInt(object1), this.entryIds.getInt(object2));
    }
```

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/client/particle/Particle.mapping (hunk 1 above)
Comment (asiekierka) [reply mappings/net/minecraft/client/particle/Particle.mapping:9]: I'm not sure I agree with posX -> x, etc.; Entity uses posX still, no?

Comment (Runemoro) [reply mappings/net/minecraft/client/particle/Particle.mapping:9]: No, entity and most other thinfs use x y z.

**Comment by modmuss50** [comment]
Updating, any further comments should go in an issue or PR

**Pull request merged**